src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs

```diff
@@ -54,6 +56,14 @@ public PostgreSqlQueryExecutor(PostgreSqlQueryConfiguration configuration, Akka.
                     {Configuration.SequenceNrColumnName} BIGINT NOT NULL,
                     CONSTRAINT {Configuration.MetaTableName}_pk PRIMARY KEY ({Configuration.PersistenceIdColumnName}, {Configuration.SequenceNrColumnName})
                 );";
+            
+            DeleteBatchSql = $@"
+                DELETE FROM {Configuration.FullJournalTableName}
+                WHERE {Configuration.PersistenceIdColumnName} = @PersistenceId AND {Configuration.SequenceNrColumnName} <= @ToSequenceNr;";
+
+            DeleteBatchSqlMetadata = $@"
```

Comment:
Can you add a comment here, explaining why we need to separate the SQL script into two?